### PR TITLE
mail: restrict staff@ and hpc-users@ to staff

### PR DESCRIPTION
**Pull request by cg505**
this might be possible to simplify, i don't know much about postfix

### CODE REVIEW

File: modules/ocf_mail/files/site_ocf/aliases [hunk 1]
```diff
@@ -32,7 +32,7 @@ site-manager: sm
 owner-sm: officers
 
 staff: staff-current
-wheel: staff-current
+wheel: officers
```
Comment (dkess) [inline modules/ocf_mail/files/site_ocf/aliases]: Why did you change this?

Comment (cg505) [reply modules/ocf_mail/files/site_ocf/aliases]: It doesn't make sense that owner-*@ all goes to staff@ in my opinion, especially as staff grows. Also, it would mean that all of those addresses would need to be specified in `protected-destinations`.

Comment (cg505) [reply modules/ocf_mail/files/site_ocf/aliases]: http://www.postfix.org/aliases.5.html ctrl-f owner-name

we might want to change this to root actually

Comment (kpengboy) [reply modules/ocf_mail/files/site_ocf/aliases]: Traditionally, we have used `wheel` as a mailing list for technical discussion among staff. Unlike root@, it was not polluted with alerts and noisy messages, and unlike officers@, it was specifically for technical discussion.

We deliberately put all staff (as opposed to, say, root staff only) on wheel@ so that none of them would be left out of technical discussion. (This was also a time when staff was a lot smaller.)

Who do you think should be on this list today? Or is this list obsolete?

Comment (cg505) [reply modules/ocf_mail/files/site_ocf/aliases]: I think it's fair to say that technical discussion general does not happen on wheel@ at this point. I didn't even know this list existed. Since staff@ is so large at this point, I don't think it makes sense to send delivery failures etc to all staff. For this I think root@ or sm@ is more reasonable.

Comment (dkess) [reply modules/ocf_mail/files/site_ocf/aliases]: In that case, you should change the `owner-*` lists to point somewhere else (like root), since we do mention [in the docs](https://www.ocf.berkeley.edu/docs/staff/mailing-lists/) that `wheel@` is a technical discussion list for staff. This should be in a separate PR since it's out-of-scope for this fix.

Comment (cg505) [reply modules/ocf_mail/files/site_ocf/aliases]: Okay. The separate PR should be merged before this one, since I haven't protected wheel@ in this PR.

Comment (cg505) [reply modules/ocf_mail/files/site_ocf/aliases]: Alternatively: we keep wheel and protect it as well, and change to owner-* lists to go to root. what do you think?

Comment (dkess) [reply modules/ocf_mail/files/site_ocf/aliases]: That sgtm

**Comment by abizer** [comment]
I think hpc-users was supposed to be an announce-only mailing list, rather than a discussion one

**Comment by cg505** [comment]
That was my impression as well. We don't want to spam that list. If desired, we could let people on the list send as well.

**Comment by cg505** [comment]
Going to merge this unless anyone opposes.
